var/spack/repos/builtin/packages/py-proj/package.py

```diff
@@ -0,0 +1,28 @@
+from spack import *
+
+class PyProj(Package):
+    """Python interface to the PROJ.4 Library."""
+    homepage = "http://jswhit.github.io/pyproj/"
+    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
+
+
+    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
+
+    # We need the benefits of this PR
+    # https://github.com/jswhit/pyproj/pull/54
+    version('citibeth-latlong2',
+        git='https://github.com/citibeth/pyproj.git',
+        branch='latlong2')
+
+    extends('python')
+
+    depends_on('py-cython')
+    depends_on('py-setuptools')
```

Comment:
Not using the proj that others are using is going to be a problem as well. There's this libproj4 with the symbol `pj_free`. A project using proj4's actual library also uses Python (ParaView is not farfetched for this) and that code loads the proj Python module. What proj library is used? If it is the one built by spack for ParaView, the Python might not match that one. ParaView can't use the one packaged here (it can't know about it), but if the soname doesn't match, you get duplicate symbols.

This really needs to use an external proj4.